src/propterm.js

```diff
@@ -275,42 +268,36 @@ $(document).ready(function () {
 
 /**
  *
- * @param code
+ * @param keyPressCode
  */
-function processKey(code) {
-    var t_str = String.fromCharCode(code);
+function processKey(keyPressCode) {
```

Comment:
This doesn't look like a refactor to remove dead code. The logic does not appear to be the same in that the test for echoKeys is no longer happening within the test for the minimum client version.

Reply:
It isn't exactly - however - the last BPC version that needed the echo trap in place has long since been deprecated, and is no longer supported in other parts of the software - this switch has been hard-coded as off/false for some time, so it wasn't doing anything anyway.

Reply:
Ok, fair enough. In future requests, let's make sure that these kinds of details are annotated for those who don't have the insight or history to these details.